src/index.js

```diff
@@ -149,6 +153,28 @@ module.exports = function(source, inputSourceMap) {
     options.sourceFileName = relative(options.sourceRoot, options.filename);
   }
 
+  if (this.version > 1 && options.presets) {
+    options.presets.forEach(preset => {
+      let name = preset;
+      let opts = {};
+
+      // if we have extra options put it in opts
+      if (Array.isArray(preset)) {
+        name = preset[0];
+        opts = preset[1];
+      }
+
+      if (
+        presetsToCheckForModules.indexOf(name) > -1 &&
+        (!opts ||
+          (opts && !opts.hasOwnProperty("modules")) ||
+          (opts && opts.modules))
+      ) {
+        this.emitWarning(new Error(modulesWarningMsg));
```

Comment:
The message can of course be improved in terms of spelling, but the `message` convention should be like above :)